bluemira/builders/EUDEMO/tf_coils.py

```diff
@@ -671,6 +675,19 @@ def _make_cas_xz(self, solid):
         outer = BluemiraFace([wires[3], wires[2]])
         self._temp_casing = [inner, outer]
 
+    def _make_centreline_koz(self, keep_out_zone):
+        """
+        Make a keep-out-zone for the TF coil centreline optimisation problem.
+        """
+        # The keep-out zone is for the TF WP centreline, so we need add to it to prevent
```

Comment:
```suggestion
        # The keep-out zone is for the TF WP centreline, so we need to add to it to prevent
```
```suggestion
        # The keep-out zone is for the TF WP centreline, so we need add to it to prevent
```